src/visor/rpc.go

```diff
@@ -20,7 +21,7 @@ func NewTransactionResult(tx *Transaction) (*TransactionResult, error) {
 		return nil, nil
 	}
 
-	rbTx, err := NewReadableTransaction(tx)
+	rbTx, err := NewReadableTransaction(tx, make([]*historydb.UxOut, 0, 0))
```

Comment:
The objective of this pr was not to fix the rpc api. However, due to the changes made in readable.go, it was necessary to send an empty array when calling `NewReadableTransaction`. This is the reason of the “Important” note in the initial pr message.